frontend/app/routes/_gcweb-app.letters._index.tsx

```diff
@@ -14,46 +16,61 @@ export const handle = {
    */
   const url = new URL(request.url);
-  const sort = url.searchParams.get('sort') || 'DESC';
+  const sort = url.searchParams.get('sort') ?? defaultSortOption;
 
   // TODO replace with actual data
   const letters = [
-    { id: '112029', subject: 'Letter subject text 1', dateSent: '2022-07-20', referenceId: '898815' },
-    { id: '014095', subject: 'Letter subject text 2', dateSent: '2023-04-22', referenceId: '443526' },
-    { id: '428715', subject: 'Letter subject text 3', dateSent: '2022-08-21', referenceId: '803445' },
-    { id: '780806', subject: 'Letter subject text 4', dateSent: '2022-02-14', referenceId: '445545' },
-    { id: '762502', subject: 'Letter subject text 5', dateSent: '2023-05-14', referenceId: '746995' },
-    { id: '295592', subject: 'Letter subject text 6', dateSent: '2022-05-04', referenceId: '766722' },
-    { id: '284742', subject: 'Letter subject text 7', dateSent: '2022-06-23', referenceId: '978920' },
-    { id: '157691', subject: 'Letter subject text 8', dateSent: '2022-04-15', referenceId: '547667' },
-    { id: '816041', subject: 'Letter subject text 9', dateSent: '2023-07-07', referenceId: '726700' },
-    { id: '538967', subject: 'Letter subject text 10', dateSent: '2023-03-11', referenceId: '624957' },
+    { id: '112029', subject: 'Letter subject text 1', dateSent: '2022-07-20T03:04:05.000Z', referenceId: '898815' },
+    { id: '014095', subject: 'Letter subject text 2', dateSent: '2023-04-22T03:04:05.000Z', referenceId: '443526' },
+    { id: '428715', subject: 'Letter subject text 3', dateSent: '2022-08-21T03:04:05.000Z', referenceId: '803445' },
+    { id: '780806', subject: 'Letter subject text 4', dateSent: '2022-02-14T03:04:05.000Z', referenceId: '445545' },
+    { id: '762502', subject: 'Letter subject text 5', dateSent: '2023-05-14T03:04:05.000Z', referenceId: '746995' },
+    { id: '295592', subject: 'Letter subject text 6', dateSent: '2022-05-04T03:04:05.000Z', referenceId: '766722' },
+    { id: '284742', subject: 'Letter subject text 7', dateSent: '2022-06-23T03:04:05.000Z', referenceId: '978920' },
+    { id: '157691', subject: 'Letter subject text 8', dateSent: '2022-04-15T03:04:05.000Z', referenceId: '547667' },
+    { id: '816041', subject: 'Letter subject text 9', dateSent: '2023-07-07T03:04:05.000Z', referenceId: '726700' },
+    { id: '538967', subject: 'Letter subject text 10', dateSent: '2023-03-11T03:04:05.000Z', referenceId: '624957' },
   ];
 
   letters.sort((a, b) => {
-    return sort === 'ASC' ? new Date(a.dateSent).getTime() - new Date(b.dateSent).getTime() : new Date(b.dateSent).getTime() - new Date(a.dateSent).getTime();
+    return sort === sortOptions.ASCENDING ? new Date(a.dateSent).getTime() - new Date(b.dateSent).getTime() : new Date(b.dateSent).getTime() - new Date(a.dateSent).getTime();
```

Comment:
```suggestion
    // prettier-ignore
    return sort === sortOptions.ASCENDING
      ? new Date(a.dateSent).getTime() - new Date(b.dateSent).getTime()
      : new Date(b.dateSent).getTime() - new Date(a.dateSent).getTime();
```

(for readability)
(ps: I hate how prettier formats *every* ternary expression)